### DISCUSSION TITLE
How to fetch input from base Query inside nested query fetcher

### DISCUSSION THREAD

**Discussion by VitalijF** [Netflix/dgs-framework#1553] [General]
Schema definition
```
baseQuery(input: BaseInput) : BaseResponse

type BaseResponse {
   nestedField: String
}

input: BaseInput {
  ids: [String!]!
}
```

**Question**: How to take base input inside nestedQuery data fetcher

**Comment by srinivasankavitha** [comment]
You can pass context around from parent to child fetchers: https://netflix.github.io/dgs/advanced/context-passing/
Hope that helps.
